JitsiConference.js

```diff
@@ -476,7 +481,7 @@ JitsiConference.prototype._init = function(options = {}) {
             userName: config.statisticsDisplayName ? config.statisticsDisplayName : this.myUserId(),
             confID: config.confID || `${this.connection.options.hosts.domain}/${this.options.name}`,
             siteID: config.siteID,
-            roomName: this.options.name,
```

Comment:
Why are you changing this?

Reply:
My bad that should be roomName. Initially I used roomName = options.name, but then I notice that it is mention as conference name in some places and room name on others so renamed all occurrence and this code was corrupted (